xdc-plugin/Makefile

```diff
@@ -1,30 +1,10 @@
-CXX = $(shell yosys-config --cxx)
-CXXFLAGS = $(shell yosys-config --cxxflags)
-LDFLAGS = $(shell yosys-config --ldflags)
-LDLIBS = $(shell yosys-config --ldlibs)
-PLUGINS_DIR = $(shell yosys-config --datdir)/plugins
-
-OBJS = xdc.o
+NAME = xdc
+SOURCES = xdc.cc
+include ../Makefile_plugin.common
 VERILOG_MODULES = BANK.v
 
-xdc.so: $(OBJS)
-	$(CXX) $(CXXFLAGS) $(LDFLAGS) -shared -o $@ $^ $(LDLIBS)
-
-install_plugin: xdc.so
-	mkdir -p $(PLUGINS_DIR)
-	cp $< $(PLUGINS_DIR)/$<
-
 install_modules: $(VERILOG_MODULES)
 	mkdir -p $(PLUGINS_DIR)/fasm_extra_modules/
 	cp $< $(PLUGINS_DIR)/fasm_extra_modules/$<
 
-test:
-	$(MAKE) -C tests all
-
-.PHONY: install
-install: install_modules install_plugin
-
-clean:
-	rm -f *.d *.o xdc.so
-	$(MAKE) -C tests clean
-
+install: install_modules
```

Comment:
I believe this will make `make` made about the double definition of `install`.  Should use `install::` instead of `install:` in all instances?  As an alternative, you could create a variable like "EXTRA_INSTALL_TARGETS" or something.

Reply:
I was also worried about that, but as it turned out without a reason. No warning and it does what it is supposed to

Reply:
Ok, cool, lets leave as is.